src/components/Nav/NavList/NavList.js

```diff
@@ -145,17 +154,25 @@ NavList.defaultProps = {
   onListClick: () => {},
   tabIndex: 0,
   title: '',
+  hasIcon: false,
```

Comment:
a better strategy is to simply use a render prop. This way you only need a single prop called renderIcon and you can perform your conditional using that. 
```
          <Icon
            className={`${navItemNamespace}__img`}
            renderIcon={this.props.renderIcon}
          />
          ```